### PR TITLE
Feature/v10/194 cleanup tasks

### PR DESCRIPTION
**Pull request by jryanconklin**
The PR addresses cleanup tasks:
* Update Demos to Use Extends for Grid
* Update Classes to Use `classes` and `other_classes` with no whitespace
* Update Large Image Style to Constrain with Bootstrap Container
* Remove `site-container.twig` template in favor of `homepage.twig` template.
* Revise ID sets for Carousel and Accordion (moving toward pa11y testing)

### CODE REVIEW

File: source/_patterns/01-atoms/badge/demo/badges.twig [hunk 1]
```diff
@@ -1,87 +1,86 @@
...
-          button_text: 'Notifications',
+  {% block column_1 %}
+    <h1>Badges</h1>
+    {% include '@molecules/card/_card-helper.twig' with {
+      card_text: badges_helper,
+      card_background: 'secondary',
+      text_color: 'light',
+      button: {
+        button_link: 'https://getbootstrap.com/docs/4.0/components/badge/',
+      }
```
Comment (illepic) [inline source/_patterns/01-atoms/badge/demo/badges.twig]: Trailing commas.

File: source/_patterns/01-atoms/list-group/demo/list-groups.twig [hunk 2]
```diff
@@ -4,204 +4,203 @@
...
+          'justify-content-between',
+          'align-items-center',
+        ] %}
+        {% set badge_items = [] %}
+        {% for item in list_group_text.items %}
+          {% set temp_item = item | merge({item_color: random(colors)}) | merge({item_badge: badge_data}) | merge({item_classes: badge_classes}) %}
+          {% set badge_items = badge_items|merge([temp_item]) %}
+        {% endfor %}
+        {% include '@atoms/list-group/_list-group.twig' with {
+          items: badge_items
```
Comment (illepic) [inline source/_patterns/01-atoms/list-group/demo/list-groups.twig]: Trailing comma.

File: source/_patterns/01-atoms/list-group/demo/list-groups.twig [hunk 3]
```diff
@@ -4,204 +4,203 @@
...
-            button_link: 'https://getbootstrap.com/docs/4.0/components/list-group/#via-javascript',
-          }
+    {#Toggleable Items#}
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Toggleable Items',
+        card_text: list_group_helper_toggleable,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/components/list-group/#via-javascript',
+        }
```
Comment (illepic) [inline source/_patterns/01-atoms/list-group/demo/list-groups.twig]: Trailing comma.

File: source/_patterns/02-molecules/card/demo/cards.twig [hunk 4]
```diff
@@ -1,135 +1,134 @@
...
+        text_color: 'light',
+      } %}
+    </div>
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Text',
+        card_text: helper_card_text,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/layout/grid/#how-it-works',
+        }
```
Comment (illepic) [inline source/_patterns/02-molecules/card/demo/cards.twig]: Trailing comma.

File: source/_patterns/02-molecules/card/demo/cards.twig [hunk 5]
```diff
@@ -1,135 +1,134 @@
...
-          {% include '@molecules/card/_card.twig' with card_width_2 %}
-          {% include '@molecules/card/_card.twig' with card_width_3 %}
-        </div>
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Width',
+        card_text: helper_card_width,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/layout/grid/#how-it-works',
+        }
```
Comment (illepic) [inline source/_patterns/02-molecules/card/demo/cards.twig]: Trailing comma.

File: source/_patterns/02-molecules/card/demo/cards.twig [hunk 6]
```diff
@@ -1,135 +1,134 @@
...
-        {% include '@molecules/card/_card.twig' with bottom_image %}
-        {% include '@molecules/card/_card.twig' with overlay_image %}
-      </div>
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Image',
+        card_text: helper_card_image,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/components/card/#card-decks',
+        }
```
Comment (illepic) [inline source/_patterns/02-molecules/card/demo/cards.twig]: Trailing comma.

File: source/_patterns/02-molecules/card/demo/cards.twig [hunk 7]
```diff
@@ -1,135 +1,134 @@
...
-        {% include '@molecules/card/_card.twig' with footer_card %}
-        {% include '@molecules/card/_card.twig' with list_card %}
-      </div>
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Structure',
+        card_text: helper_card_structure,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/components/card/#header-and-footer',
+        }
```
Comment (illepic) [inline source/_patterns/02-molecules/card/demo/cards.twig]: Trailing comma.

File: source/_patterns/02-molecules/card/demo/cards.twig [hunk 8]
```diff
@@ -1,135 +1,134 @@
...
-    {% endblock column_1 %}
-  {% endembed %}
-</div>
+    <div class="mb-3">
+      {% include '@molecules/card/_card-helper.twig' with {
+        card_title: 'Background and Border',
+        card_text: helper_card_colors,
+        button: {
+          button_link: 'https://getbootstrap.com/docs/4.0/components/card/#card-columns',
+        }
```
Comment (illepic) [inline source/_patterns/02-molecules/card/demo/cards.twig]: Trailing comma.

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 9]
```diff
@@ -1,111 +1,112 @@
...
-        </div>
+  {% block column_1 %}
+    <h1>Navs</h1>
+    {% include '@molecules/card/_card-helper.twig' with {
+      card_text: navs_helper,
+      card_background: 'secondary',
+      text_color: 'light',
+      button: {
+        button_link: 'http://getbootstrap.com/docs/4.0/components/navs/',
+      }
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Trailing comma.

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 10]
```diff
@@ -1,111 +1,112 @@
...
-        <h1>Pills</h1>
-        {% include '@molecules/nav/_nav.twig' with {
-          nav_pill: 'true',
-        } %}
-      </div>
+    </div>
+    <div class=" mb-3">
+      <h1>Vertical Alignment</h1>
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_vertical: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Why `true` as a string and not a boolean?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 11]
```diff
@@ -1,111 +1,112 @@
...
-        <h1>Fill</h1>
-        {% include '@molecules/nav/_nav.twig' with {
-          nav_pill: 'true',
-          nav_fill: 'true',
-        } %}
-      </div>
+    <div class=" mb-3">
+      <h1>Tabs</h1>
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_tabs: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Should this be boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 12]
```diff
@@ -1,111 +1,112 @@
...
-        <h1>Justify</h1>
-        {% include '@molecules/nav/_nav.twig' with {
-          nav_pill: 'true',
-          nav_justify: 'true',
-        } %}
-      </div>
+    <div class=" mb-3">
+      <h1>Pills</h1>
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_pill: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 13]
```diff
@@ -1,111 +1,112 @@
...
-        {% include '@molecules/nav/_nav.twig' with {
-          nav_pill: 'true',
-          nav_other_classes: 'flex-column flex-sm-row',
-          nav_items: responsive_items,
-        } %}
-      </div>
+    <div class=" mb-3">
+      <h1>Fill</h1>
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_pill: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 14]
```diff
@@ -1,111 +1,112 @@
...
+        nav_fill: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 15]
```diff
@@ -1,111 +1,112 @@
...
-        {% include '@molecules/nav/_nav.twig' with {
-          nav_element: 'ul',
-          nav_tabs: 'true',
-          nav_items: nav_items_dropdown,
-        } %}
-      </div>
+    <div class=" mb-3">
+      <h1>Justify</h1>
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_pill: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 16]
```diff
@@ -1,111 +1,112 @@
...
+        nav_justify: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/02-molecules/nav/demo/navs.twig [hunk 17]
```diff
@@ -1,111 +1,112 @@
...
-        </div>
+    <div class=" mb-3">
+      <h1>Responsive nav</h1>
+      {% set responsive_items = [] %}
+      {% for item in nav_items_responsive %}
+        {% set temp_item = item | merge({nav_item_element: 'a'}) %}
+        {% set responsive_items = responsive_items|merge([temp_item]) %}
+      {% endfor %}
+      {% include '@molecules/nav/_nav.twig' with {
+        nav_pill: 'true',
```
Comment (illepic) [inline source/_patterns/02-molecules/nav/demo/navs.twig]: Boolean or string?

File: source/_patterns/03-organisms/accordion/_accordion.twig [hunk 18]
```diff
@@ -14,6 +14,8 @@
   * https://getbootstrap.com/docs/4.0/components/card/
 #}
 
+{% set accordion_id = random(9999) %}
```
Comment (illepic) [inline source/_patterns/03-organisms/accordion/_accordion.twig]: Use `random()` with no args for now. I want to get feedback on the screenreader experience here first. Later we can set a global var to seed this, ie something like `random(ACCESSIBILITY_ID_MAX)` set in PL, Drupal, and Grav.

File: source/_patterns/03-organisms/navbar/demo/navbars.twig [hunk 19]
```diff
@@ -1,17 +1,18 @@
...
-{% endembed %}
+  {% block column_1 %}
+    <h1>Navbar</h1>
+    {% include '@molecules/card/_card-helper.twig' with {
+      card_text: navbar_helper,
+      card_background: 'secondary',
+      text_color: 'light',
+      button: {
+        button_link: 'http://getbootstrap.com/docs/4.0/components/navbar/',
+      }
```
Comment (illepic) [inline source/_patterns/03-organisms/navbar/demo/navbars.twig]: Trailing comma.

File: source/_patterns/01-atoms/dropdown/_dropdown.twig [hunk 20]
```diff
@@ -24,15 +27,16 @@
   {% set dropdown_class = 'dropdown' %}
 {% endif %}
 
-{% set classes = [
+{% set dropdown_classes = [
   dropdown_class,
   dropup ? 'dropup',
-] | sort | join(' ') | trim ~ ' ' ~ dropdown_other_classes %}
+  dropdown_other_classes,
```
Comment (tjheffner) [inline source/_patterns/01-atoms/dropdown/_dropdown.twig]: I get why we might specifically scope classes to a thing, like `badge_classes` or `button_classes`, but do we need all these extra undefined(?) `button_other_classes` and `dropdown_other_classes` for anything special? 

twig's `|merge` filter should be fine for passing more classes. it looks like `|merge` is being used for exactly this already in this pr, in the case of action group list items

Comment (jryanconklin) [reply source/_patterns/01-atoms/dropdown/_dropdown.twig]: For many of the examples we're not sending in an array to merge and sending a string causes it to kersplode. I can remove the cruft / unused ones for sure, but fwiw I think doing it this way is easier to catch what it's doing at a glance.

**Comment by jryanconklin** [comment]
@illepic Updated!

File: source/_patterns/01-atoms/dropdown/_dropdown.twig (hunk 20 above)
Comment (illepic) [reply source/_patterns/01-atoms/dropdown/_dropdown.twig]: @tjheffner , I'm going to merge this. Let's make a ticket to address what you brought up.

**Pull request merged**